package.json

```diff
@@ -71,6 +71,7 @@
     "preact": "*"
   },
   "devDependencies": {
+    "@material/chips": "^0.31.0",
```

Comment:
Shouldn't this be in `dependencies` and not `devDependencies`?

Reply:
oh yes